src/openrct2/localisation/Language.h

```diff
@@ -75,6 +75,7 @@ struct language_descriptor
     const char* locale;
     const utf8* english_name;
     const utf8* native_name;
+    uint8_t fallback;
```

Comment:
Will the fallbacks be defined in a follow-up PR? That was a bit unclear to me.

Reply:
None of the current languages need a fallback. This PR introduces the mechanism so that new languages (e.g. Canadian French) can take advantage of it.